data/human/culture conversations.txt

```diff
@@ -80,15 +80,44 @@ outfit "Mug"
 
 
 
+mission "New Holland Foibles"
+	minor
+	source "New Holland"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`The smell of burning metal fills the air as you walk down the uncrowded streets of the factory town near the spaceport, keeping away from the arterial supply route directly feeding the port and its hangars.`
+			`	Despite the distance and absent traffic though, it is far from quiet. Several factory buildings make discernible thrumming noises, and grinders echo amidst the silence as you investigate. Turning a corner, you come across an alley running along a lengthy chain link fence line which obstructs your path to anything interesting. Beyond it, you can see a throng of men cutting away at a rather well kept spaceship. Parts regularly fall to the floor below, apparently heedless of those working below.`
```

Comment:
I don't overly like just "obstructs your path" - I in fact explicitly adapted from that in the original piece. It didn't make sense to me that you would walk down a street and just be blocked by a fenced open area.
"their companions" also feels unnecessary but eh

Reply:
I agree that with both. Their companions is unnecessary and the rest just need approval.

Reply:
```suggestion
			`	Despite the distance and absent traffic though, it is far from quiet. Several factory buildings make discernible thrumming noises, and grinders echo amidst the silence as you investigate. Turning a corner, you come across an alley running along a lengthy chain link fence line. Beyond it, you can see a throng of men cutting away at a spaceship. Parts regularly fall to the floor below, the workers apparently heedless of those below.`
```
How about this? I might have misunderstood the scene and where the fence is oriented relative to the alley.

Reply:
Not entirely sure of the worker part. @MasterOfGrey?

Reply:
>  I might have misunderstood the scene and where the fence is oriented relative to the alley.

The idea being you walk down the alley, and at some point it is no longer hemmed in by buildings on either side. It is instead just fenced on one side. The fence blocks your access to the things next to the otherwise unremarkable alley.

Reply:
Gotcha, makes sense to me.

> Not entirely sure of the worker part.

That's to fix what I mentioned above:

>My main problem is just as it was, apparently heedless seemed to refer to the ship parts themselves, which are inanimate objects.

Including `the workers` makes it clear it's the workers who are heedless, not the ship parts.

Reply:
It works. Just need to edit out the repeated "below" and it _should_ be fine.

Reply:
Oops, I gotcha.
```suggestion
			`	Despite the distance and absent traffic though, it is far from quiet. Several factory buildings make discernible thrumming noises, and grinders echo amidst the silence as you investigate. Turning a corner, you come across an alley running along a lengthy chain link fence line. Beyond it, you can see a throng of men cutting away at a spaceship. Parts regularly fall to the floor, the workers apparently heedless of those below.`
```
@MasterOfGrey look good to you as well?